Gemfile

```diff
@@ -46,12 +46,14 @@ gem 'role_playing', github: 'jnv/role_playing'
 gem 'logging'
 gem 'oauth2', github: 'cvut/oauth2', ref: 'd9f6fc3'
 gem 'celluloid'
+gem 'rack-timeout'
 
 ## For console
 gem 'pry', require: false
 
 ## Error reporting
 gem 'sentry-raven'
+gem 'rollbar'
```

Comment:
This is required by Pliny in newer versions. Without this it would not want to load. Maybe we can somehow patch Pliny to avoid this requirement.